<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_movie</class>
 <widget class="QWidget" name="admin_movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华康方圆体W7</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>电影管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sdufe.qrc">
    <normaloff>:/山财大校徽.png</normaloff>:/山财大校徽.png</iconset>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>571</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="query">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电影信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>320</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>增加空白行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回列表</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>586</x>
     <y>320</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择查询方式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按名称查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按类型查询</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>返回管理员主界面</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sdufe.qrc"/>
 </resources>
 <connections/>
</ui>
